autogen/docs/behaviorspace.md.mustache

```diff
@@ -83,7 +83,11 @@ names will help you keep them straight.
 
 **Vary variables as follows:** This is where you specify which settings you want
 varied, and what values you want them to take. Variables can include sliders,
-switches, choosers, and any global variables in your model.
+switches, choosers, and any global variables in your model. You may notice that
+the view and plots do not correspond to the values in the widgets once the experiment is completed. 
+This is because the globals that are controlled by widgets are reset to their initial state at the end of the 
+experiment. To synchronize your view and plots with the widgets, run your procedure
+that initializes the view. 
```

Comment:
please remove the trailing whitespaces
![Screenshot 2023-08-09 at 12 00 49 PM](https://github.com/NetLogo/NetLogo/assets/56969066/cbdfaf44-caad-49e1-b144-a3601ae66e21)